src/applications/vaos/new-appointment/components/TypeOfMentalHealthPage.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect } from 'react';
+import { shallowEqual, useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import { VaRadioField } from '@department-of-veterans-affairs/platform-forms-system/web-component-fields';
+import FormButtons from '../../components/FormButtons';
+import { getFormPageInfo } from '../redux/selectors';
+import { TYPES_OF_MENTAL_HEALTH } from '../../utils/constants';
+import { focusFormHeader } from '../../utils/scrollAndFocus';
+
+import {
+  openFormPage,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+  updateFormData,
+} from '../redux/actions';
+
+const pageKey = 'typeOfMentalHealth';
+const pageTitle = 'Which type of mental health care do you need?';
+
+const initialSchema = {
+  type: 'object',
+  required: ['typeOfMentalHealthId'],
+  properties: {
+    typeOfMentalHealthId: {
+      type: 'string',
+      enum: TYPES_OF_MENTAL_HEALTH.map(type => type.id),
+    },
+  },
+};
+
+const uiSchema = {
+  typeOfMentalHealthId: {
+    'ui:title': pageTitle,
+    'ui:widget': 'radio', // Required
+    'ui:webComponentField': VaRadioField,
+    'ui:options': {
+      classNames: 'vads-u-margin-top--neg2',
+      labelHeaderLevel: '1',
+      labels: {
+        [TYPES_OF_MENTAL_HEALTH[0].id]: TYPES_OF_MENTAL_HEALTH[0].name,
+        [TYPES_OF_MENTAL_HEALTH[1].id]: TYPES_OF_MENTAL_HEALTH[1].name,
+      },
+      descriptions: {
+        [TYPES_OF_MENTAL_HEALTH[0].id]:
+          'Therapy, medication, and other services to help with posttraumatic ' +
```

Comment:
According to the design file, it's "posttraumatic" but shouldn't this be "post-traumatic"? @department-of-veterans-affairs/uae-ux